lib/Models/FunctionParameter.js

```diff
@@ -59,6 +59,12 @@ var FunctionParameter = function(options) {
     this.name = defaultValue(options.name, options.id);
 
     /**
+     * Gets or sets the display value of the parameter.
+     * @type {String}
+     */
+    this.displayValue = defaultValue(options.displayValue, '');
```

Comment:
Shouldn't this be computed from the actual value, rather than a separate property?  It's ok if it's settable, but the setter should set the underlying value.